static/img/devcycle-no-fill.svg

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 25.4.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Logo" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 100 109.6" style="enable-background:new 0 0 100 109.6;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#101827;}
+</style>
+<g id="Togglebot">
+	<path id="Thicker_Togglebot" class="st0" d="M50.1,109.6c-6.5,0-12.5-3.7-15.4-9.5c-0.4-0.7,0-1.6,0.7-2h0.1l3.3-1.3
```

Comment:
```suggestion
<g id="Togglebot">
	<path id="Thicker_Togglebot" d="M50.1,109.6c-6.5,0-12.5-3.7-15.4-9.5c-0.4-0.7,0-1.6,0.7-2h0.1l3.3-1.3
```

Removing the fill from the SVG will make this logo work in dark and light modes. It will also allow us to add a subtle hover effect.

By the way, can we add the DevCycle logo to the main landing page once this PR is merged?